wiki/People/Beatmap_Nominators/Rules/en.md

```diff
@@ -25,7 +25,14 @@ Violations of those rules may be reported to a member of the [Quality Assurance
 + **Do not nominate maps with unrankable issues.** The General Ranking Criteria and additionally all mode-specific Ranking Criteria apply to all sets with mode-specific difficulties.
 + **Do not qualify mapsets whose song already has a Qualified mapset, unless no game modes of both sets overlap - this includes remixed versions**. For example an osu!taiko and osu! mapset of the same song can exist in the Qualified section, whilst an osu!taiko and osu!/osu!taiko hybrid set cannot. This ensures variety among Ranked beatmaps within their respective game modes.
 + **If a bubbled map contains unrankable issues, its bubble must be popped.** This applies even if you do not intend to nominate the mapset.
-+ **At least 24 hours must pass between a first nomination and a qualification on every mapset, ignoring renominations for minor changes.** This gives other Beatmap Nominators and also the wider community the chance to provide additional suggestions before qualification and veto the placed nomination if necessary.
++ **At least 24 hours must pass between a first nomination and a qualification on every beatmap set if...**
+	+	the beatmap set has received no nominations prior. 
```

Comment:
use spaces instead of tabs

Reply:
going to include any reasoning after "the beatmap set has received no nominations prior."??

Reply:
idk it seems like a reasoning to help people understand why would be helpful.